app/api/models/statistics.py

```diff
@@ -112,8 +112,9 @@ class GenomeStatistics(BaseModel):
 
     def __init__(self, **data):
         data["_compiled_data"] = {
-            stats_item["name"]: stats_item["statisticValue"]
-            for stats_item in data["_raw_data"]
+            si["name"]: si["statisticValue"]
```

Comment:
I would keep `stats_item` for better context if we have space (code is on 3 lines anyway) but that's  just a matter of taste.